vpr/src/place/place.cpp

```diff
@@ -2656,6 +2674,26 @@ static void print_resources_utilization() {
     VTR_LOG("\n");
 }
 
+static void init_annealing_state(t_annealing_state* state,
+                                 const t_annealing_sched& annealing_sched,
+                                 float t,
+                                 float rlim,
+                                 int move_lim_max,
+                                 float crit_exponent) {
+    state->alpha = annealing_sched.alpha_min;
+    state->t = t;
+    state->restart_t = t;
+    state->rlim = rlim;
+    state->inverse_delta_rlim = 1 / (rlim - FINAL_RLIM);
+    state->move_lim_max = std::max(1, move_lim_max);
+    if (annealing_sched.type == DUSTY_SCHED) {
```

Comment:
If the schedule is moved behind and interface, there should be another call here.

Reply:
Agreed, but I want to minimize changes for now.